src/Data/MASA.Utils.Data.Elasticsearch/_Imports.cs

```diff
@@ -0,0 +1,20 @@
+global using Elasticsearch.Net;
```

Comment:
See the `Unchanged files with check annotations` and fix them.

Reply:
Issues here will be resolved on other branches later